cypress/e2e/po/pages/extensions.po.ts

```diff
@@ -228,6 +229,14 @@ export default class ExtensionsPagePo extends PagePo {
     return this.extensionTabs.clickNthTab(4);
   }
 
+  extensionTabBuiltinClick(): Cypress.Chainable {
+    return this.self().get('[data-testid="btn-builtin"]').click();
+  }
+
+  extensionTabBuiltin() {
+    return new ComponentPo('[data-testid="builtin"]', this.self());
```

Comment:
this should probably be a new fn in `TabbedPo`
```
getTab(name) {
  return this.self().find([data-testid="name"])
}
```

Reply:
Addressed